pallets/polkadex-ido/src/lib.rs

```diff
@@ -231,13 +237,14 @@ pub mod pallet {
             let token_a_price_per_token_b_perquintill = Perbill::from_rational(token_a_price_per_token_b, 1_000_000_000_000_u128.saturated_into());
             ensure!(!token_a_price_per_token_b_perquintill.is_zero(), <Error<T>>::PricePerTokenCantBeZero);
             ensure!(start_block < close_round_block, <Error<T>>::StartBlockMustBeLessThanEndblock);
-            // TODO: This block of code handling unwrap will be updated when working on vesting module
+            ensure!(vesting_per_block <= 100, <Error<T>>::NotAllowed);
```

Comment:
Where from this `100` get here? shouldn't this be defined as a some config or configurable argument and not just hardcoded value?

Reply:
@felixfaisal What's the reasoning behind 100?

Reply:
@Gauthamastro  @serhii-temchenko  So we cannot directly give a percentage to the explorer, The idea here is the user gives a number between 1 to 100 as percentage.